.kokoro/nightly/create-versions-csv.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+#input -> text file that lists versionless coordinates (libraries.txt)
+# this libraries.txt will be generated on every run of the script.
```

Comment:
The script for the job should not expect input.

Reply:
appropriate comments added.